monitoring/tests/system/test_vpcsc_v3.py

```diff
@@ -386,17 +335,13 @@ def test_create_notification_channel(self):
         PROJECT_OUTSIDE is None,
         reason="Missing environment variable: GOOGLE_CLOUD_TESTS_VPCSC_OUTSIDE_PERIMETER_PROJECT",
     )
-    def test_delete_notification_channel(self):
-        client = monitoring_v3.NotificationChannelServiceClient()
-        name_inside = client.notification_channel_path(
-            PROJECT_INSIDE, "mock_notification_channel"
-        )
-        delayed_inside = lambda: client.delete_notification_channel(name_inside)
-        name_outside = client.notification_channel_path(
-            PROJECT_OUTSIDE, "mock_notification_channel"
-        )
-        delayed_outside = lambda: client.delete_notification_channel(name_outside)
-        TestVPCServiceControlV3._do_test(delayed_inside, delayed_outside)
```

Comment:
~~Why does this PR delete this test?~~ I see that it is moved below:  is there a reason for reordering the testcases here?